pygsti/circuits/gstcircuits.py

```diff
@@ -528,7 +527,7 @@ def add_to_plaquettes(pkey_dict, plaquette_dict, base_circuit, maxlen, germ, pow
     if nest:
         #keep track of running quantities used to build circuit structures
         running_plaquette_keys = {}  # base-circuit => (maxlength, germ) key for final plaquette dict
-        running_plaquettes = _collections.OrderedDict()  # keep consistent ordering in produced circuit list.
+        running_plaquettes = dict()  # keep consistent ordering in produced circuit list.
```

Comment:
Just a general note that I've noticed several of these OrderedDict -> dict changes in recent PRs. It's not a problem, but just wanted to link it back to our discussion in #427 where we wanted to annotate dicts where order is explicitly important (e.g. where keys/values are iterated over with an expected order later).
I only bring this up here since "keeping consistent ordering" is part of the comment for this dict instantiation - is this one such place where order matters, and we should finalize our decision from #427?